auction/program/src/processor/create_auction.rs

```diff
@@ -160,13 +215,16 @@ pub fn create_auction(
         gap_tick_size_percentage: args.gap_tick_size_percentage,
         instant_sale_price,
         name,
+        decrease_rate: Some(send_decrease_rate),
+        decrease_interval: Some(decline_interval),
+        auction_start_time: None,
     }
     .serialize(&mut *accounts.auction_extended.data.borrow_mut())?;
 
     // Configure Auction.
     AuctionData {
         authority: args.authority,
-        bid_state: bid_state,
```

Comment:
Latest auction contract includes `bid_state: bid_state,`. I'm guessing this is code cleanup to the same assignment logic

Reply:
Yes, it's the preferred way of submitting an arg that has the same name as the var submitted